R/utility-functions-analysis.R

```diff
@@ -0,0 +1,165 @@
+# utility-functions-analysis.R
+#
+# utility functions for analysis
+# pulled out of `bayesian.R` because they're useful
```

Comment:
I don't think we need to mention where these come from since git will track that. Maybe just "utility functions for analysis. can be used with both bayesian.R and objectivefun.R"